app/views/govuk_publishing_components/components/_step_by_step_nav.html.erb

```diff
@@ -77,8 +77,10 @@
             <%
               in_substep = false
               options[:step_nav_content_id] = step_nav_content_id
+              options[:step_title] = step[:title]
               options[:step_index] = step_index
               options[:link_index] = 0
+              options[:ga4_tracking] = ga4_tracking
```

Comment:
Not sure where to put this comment. Can you expand the YML file example of with GA4 tracking to include links? That would show the extra attributes that are added to links when this option is enabled.

Reply:
Thanks @andysellick, I've done this now.